src/main/java/com/github/webicitybrowser/spec/fetch/FetchConsumeBodyAction.java

```diff
@@ -1,7 +1,9 @@
 package com.github.webicitybrowser.spec.fetch;
 
 public interface FetchConsumeBodyAction {
-	
+
+
 	void execute(FetchResponse response, boolean success, byte[] body);
+	FetchResponse getFetchResponse();
```

Comment:
Also, this method should not exist, as this interface is meant to represent an algorithm that can be called, and should not be stateful
For testing purposes, you can use Mockito and make a mock, and then use Mockito.verify to ensure it gets called once, and with the correct parameters

Reply:
Yes i did that to somehow get fetch response in test. I didn't use Mockito earlier so i didn't knew i can do that.